test/test_torch.py

```diff
@@ -12504,6 +12506,62 @@ def binary_check_mem_overlap(self, inplace_op, value=-0.5, device='cpu'):
         with self.assertRaisesRegex(RuntimeError, 'single memory location'):
             inplace_op(tensor, other)
 
+    @staticmethod
+    def binary_check_input_output_mem_overlap(self, data, sz, op, device):
```

Comment:
device is unused, do you need to pass it

Reply:
addressed